packages/central-server/pm2.config.cjs

```diff
@@ -1,15 +1,20 @@
 const os = require('node:os');
+
+const cwd = '.'; // IMPORTANT: Leave this as-is, for production build
```

Comment:
@passcod I haven't been able to work out exactly how this value is changed on production. It's automatically done by the `bestool`?

Reply:
https://github.com/beyondessential/tamanu/blob/main/.github/workflows/cd-package-servers.yml#L44-L52